app/views/admin/claims/show.html.erb

```diff
@@ -14,9 +14,14 @@
 
     <%= button_to "Approve",
                   admin_claim_checks_path(@claim),
-                  method: :post,
                   params: { result: "approved" },
                   class: "govuk-button",
                   data: { confirm: "Are you sure you want to approve this claim?" } %>
+
+    <%= button_to "Reject",
+                admin_claim_checks_path(@claim),
+                params: { result: "rejected" },
+                class: "govuk-button",
```

Comment:
Should we make this a secondary button? https://design-system.service.gov.uk/components/button/#secondary-buttons

Reply:
I guess right now we are not trying to solve the UI, as long as we can reject and approve a claim we can come back and resolve how that is achieved at a later date, there is work on that happening already.

I'm not sure how I feel about reject being secondary as well, that is a user performing the check doesn't know the outcome going into the check?

Hapyy to change this though if we think it is the right call to make right now.

Reply:
Yeah I think it's a quick win to have the button visually different. We aim to follow the design system and it says "Avoid using multiple default buttons on a single page". Not sure which one to use though

Reply:
Also, happy to see this addressed later